### DISCUSSION TITLE
工具人團契，軟性話題投票

### DISCUSSION THREAD

**Discussion by schwannden** [wanlong-church/service#22] [General]
直接在此留言你想討論的話題，並且給其他你看到想討論的話題點個讚

**Comment by schwannden** [comment]
你怎樣定義一份好的工作與生活平衡的工作？

**Reply by 0x56iceroy** [reply]
工作與生活可以分離而且能滿足馬斯洛需求

**Comment by schwannden** [comment]
關於一邊工作一邊成長，你有什麼方法？或者你卡關的地方在哪裡？

**Reply by 0x56iceroy** [reply]
鑽研;基礎知識不完備，時間不夠。

**Comment by 0x56iceroy** [comment]
我覺得吧，
軟工的世界還是很大，即便世界瞬息萬變不免讓人焦慮，
但若能彌補知識上的落差可以更新到最新的趨勢，也無非壞事也避免自己閉門造車。

**Comment by david83715** [comment]
分享大家的MBTI或九型(會不會太軟XD)

**Reply by lowkingshih** [reply]
感覺可以順便分享各個 MBTI 在職場上有沒有遇到什麼優勢跟困境🤣

**Comment by david83715** [comment]
你覺得自己是適合遠端工作的人嗎?遠端工作有什麼優缺點?

**Comment by lowkingshih** [comment]
如何從初級工程師走向資深工程師
